.flake8

```diff
@@ -2,4 +2,11 @@
 import-order-style = google
 application-import-names = e2c, tests
 per-file-ignores =
-    tests/*: D104
+    # D104 Missing docstring in public package
+    tests/__init__.py: D104,
+    tests/e2c/__init__.py: D104,
+ignore =
+    # E203 whitespace before ':'
+    E203,
+    # line break before binary operator
+    W503
```

Comment:
Why? I tested locally and it's passing.

Reply:
In `run_script.py`:

![Screenshot 2020-07-15 at 19 58 55](https://user-images.githubusercontent.com/19307850/87579042-9ea26d00-c6d5-11ea-9160-a8c7b3de0894.png)

Reply:
Now, it seems to work... Don't know why.